src/video_core/renderer_opengl/gl_shader_decompiler.cpp

```diff
@@ -2758,6 +2835,10 @@ class GLSLDecompiler final {
     ShaderWriter code;
 
     std::optional<u32> max_input_vertices;
+
+    std::array<bool, HelperFunctions::Total> helper_functions_enabled;
```

Comment:
This can be initialized in the class body like so:

```suggestion
    std::array<bool, HelperFunctions::Total> helper_functions_enabled{};
```

allowing the additions to the constructor initializer list to be removed. Similarly `atomic_memory_types`'s entry in the initializer list can be removed because it has its own constructor that always establishes its invariants.